ui/ops/src/dynamic/widgets/meter/MeterHistoryCard.vue

```diff
@@ -24,6 +24,16 @@
               </v-list-item>
               <v-list-subheader title="Data"/>
               <period-chooser-rows v-model:start="_start" v-model:end="_end" v-model:offset="_offset"/>
+              <v-list-subheader title="Metric" v-if="Object.keys(props.scaleValues).length > 0"/>
+              <v-list-item v-if="Object.keys(props.scaleValues).length > 0">
+                <v-btn-toggle
+                    mandatory v-model="metricType"
+                    variant="outlined" density="compact"
+                    divided class="ml-4">
+                  <v-btn :value="'unscaled'" size="small" :text="unit"/>
+                  <v-btn :value="'scaled'" size="small" :text="props.scaleUnit"/>
+                </v-btn-toggle>
+              </v-list-item>
```

Comment:
For things like this where you have two adjacent elements guarded by the same `if` you should use

```
<template v-if="...">
  <el1>
  <el2>
</template>
```